mesa/testProton.py

```diff
@@ -37,7 +38,8 @@ def test_networks(self):
         for i in range(0,10): 
             pp.Person(m)
         for i in range(0,10):
-            for l in links[i]: pp.Person.persons[i].neighbors.get('friendship').add(pp.Person.persons[l])
```

Comment:
In general, it's better if you don't make this kind of formatting change (except maybe on a dedicated commit)